include/seastar/websocket/common.hh

```diff
@@ -243,16 +243,19 @@ protected:
 
     sstring _subprotocol;
     handler_t _handler;
+    bool _is_client;
```

Comment:
It seems that `std::optional<uint32_t> _masking_key` would fit better here. Constructor and send_key() would need to be updated accordingly